json-format.md

```diff
@@ -113,11 +113,18 @@ If an implementation determines that the type of the `data` attribute is
 `Binary` or `String`, it MUST inspect the `contentType` attribute to determine
 whether it is indicated that the data value contains JSON data.
 
-If the `contentType` value is ["application/json"][RFC4627], or any media type
+If the `contentType` value is either ["application/json"][RFC4627] or any media type
 with a [structured +json suffix][RFC6839], the implementation MUST translate
 the `data` attribute value into a [JSON value][JSON-Value], and set the `data`
 member of the envelope JSON object to this JSON value.
 
+If the `contentType` value does not follow the [structured +json suffix][RFC6839]
+but still implies JSON, the implementation SHOULD translate the `data` attribute
+value into a [JSON value][JSON-Value], and set the `data` member of the envelope
```

Comment:
s/member/attribute/ just for consistency.

Reply:
Done

Reply:
thanks